.github/actions/trun/main.ts

```diff
@@ -0,0 +1,96 @@
+import * as core from "./deps/actions/core.ts"
```

Comment:
While the GH table artifact is cool, I'm not so sure utilizing actions-specific functionality in the trun CLI is merited. Better as a general CLI tool imo.

Reply:
How about we add an output flag to trun that emits the test results as csv file w/ columns name & exitcode. and then whoever is using it has the option to read  from that csv file and package it up as a gh action.

i.e. we have the option of making the action / table summary local to capi itself, _if_ we choose to.

Reply:
If you're passionate about it, an output flag is fine. Instead of a csv, how about plain JSON. The flag can be `--json` (much like `deno bench`).

Regarding the table... I don't think it adds much value.

- When one sees the check passed, they'll ignore output entirely (as there should be none).
- When one sees the check failed, they'll click into the run to see the run-specific logs (and therefore the list (not table) of failures).

Reply:
yup thats fair. -- and json it is then